pkg/hubble/cell/config.go

```diff
@@ -82,3 +82,25 @@ func getDefaultMonitorQueueSize(numCPU int) int {
 	monitorQueueSize := min(numCPU*ciliumDefaults.MonitorQueueSizePerCPU, ciliumDefaults.MonitorQueueSizePerCPUMaximum)
 	return monitorQueueSize
 }
+
+// hubbleConfigWithTLS combines hubble config and TLS config to implement HubbleConfig interface
+type hubbleConfigWithTLS struct {
+	ListenAddress    string `mapstructure:"hubble-listen-address"`
+	PreferIpv6       bool   `mapstructure:"hubble-prefer-ipv6"`
+	DisableServerTLS bool   `mapstructure:"hubble-disable-server-tls"`
+}
```

Comment:
We can omit the mapstructure annotations here as this struct won't be used to parse configuration.